src/transform/import_resolution.rs

```diff
@@ -26,10 +27,50 @@ pub fn resolve_imports<R>(
     rt: RichTerm,
     resolver: &mut R,
 ) -> Result<(RichTerm, Vec<FileId>), ImportError>
+where
+    R: ImportResolver,
+{
+    let (term, resolved, errors) = resolve_imports_resillient(rt, resolver);
+    match errors.first().cloned() {
+        Some(first) => Err(first),
+        None => Ok((term, resolved)),
+    }
+}
+
+/// Resolve the import if the term is an unresolved import, or return the term unchanged. As
+/// [`super::share_normal_form::transform_one`], this function is not recursive.
+pub fn transform_one<R>(
+    rt: RichTerm,
+    resolver: &mut R,
+    parent: &Option<PathBuf>,
+) -> Result<RichTerm, ImportError>
+where
+    R: ImportResolver,
+{
+    let term = rt.as_ref();
+    match term {
```

Comment:
Would that make sense to reuse `transform_one_resilient` here, as you did in `resolve_imports` using `resolve_imports_resilient`?

Reply:
Ah yes 👍